lib/dotcom_web/router.ex

```diff
@@ -279,6 +288,12 @@ defmodule DotcomWeb.Router do
     end
   end
 
+  scope "/", DotcomWeb do
+    pipe_through([:browser, :browser_live])
+
+    live "/demo", Live.Demo
+  end
+
```

Comment:
Can you remove this please?

Reply:
[Woops](https://github.com/mbta/dotcom/pull/2663/commits/13f238371dc720192e64a079fab6eef5e1e8aa5a)